collect_app/src/androidTest/java/org/odk/collect/android/ResetAppStateTestCase.java

```diff
@@ -70,8 +71,8 @@ public void resetSettingsTest() throws IOException {
         resetAppState(Collections.singletonList(ResetUtility.ResetAction.RESET_PREFERENCES));
 
         SharedPreferences settings = PreferenceManager.getDefaultSharedPreferences(Collect.getInstance());
-        assertEquals(null, settings.getString(PreferenceKeys.KEY_USERNAME, null));
-        assertEquals(null, settings.getString(PreferenceKeys.KEY_PASSWORD, null));
+        assertNull(settings.getString(PreferenceKeys.KEY_USERNAME, null));
```

Comment:
I don't think there's a 1-arg option for that method: https://developer.android.com/reference/android/content/SharedPreferences.html#getString(java.lang.String,%20java.lang.String)